src/modules/GroupChannel/components/MessageInputWrapper/useHandleUploadFiles.tsx

```diff
@@ -96,7 +96,7 @@ export const useHandleUploadFiles = (
       if (sendingFiles.length === 1) {
         logger.info('Channel|useHandleUploadFiles: sending one file.');
         const [file] = sendingFiles;
-        return sendFileMessage({ file, parentMessageId: quoteMessage?.messageId });
+        return sendFileMessage({ file, parentMessageId: quoteMessage?.messageId, ...params });
```

Comment:
Let's move `...params` to the most front.

Reply:
addressed in https://github.com/sendbird/sendbird-uikit-react/pull/1218/commits/474a8e55880595639d5b51f3fa6eb1340e2bd8f8